plugins/tiddlywiki/tiddlyweb/sse-server.js

```diff
@@ -0,0 +1,85 @@
+/*\
+title: $:/plugins/tiddlywiki/tiddlyweb/sse-server.js
+type: application/javascript
+module-type: route
+
+GET /events/plugins/tiddlywiki/tiddlyweb
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+var wikis = [];
+var connections = [];
+
+/*
+Setup up the array for this wiki and add the change listener
+*/
+function setupWiki(wiki) {
+	var index = wikis.length;
+	// Add a new array for this wiki (object references work as keys)
+	wikis.push(wiki);
+	connections.push([]);
+	// Listen to change events for this wiki
+	wiki.addEventListener("change",function(changes) {
+		getWikiConnections(wiki).forEach(function(item) {
+			item.emit("change",JSON.stringify(changes));
```

Comment:
I'd be inclined to take the JSON.stringify out of the loop.

Reply:
Done as well.